Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/NodePaletteModelHelper.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "NodePaletteModelHelper.h"
+#include <EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/AnimGraphPlugin.h>
+#include <EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/BlendGraphWidget.h>
+#include <EMotionFX/Source/AnimGraphObjectFactory.h>
+#include <GraphCanvas/Widgets/NodePalette/TreeItems/NodePaletteTreeItem.h>
+
+namespace EMStudio
+{
+    NodePaletteModelHelper::NodePaletteModelHelper(AnimGraphPlugin* plugin)
+        : m_rootItem(aznew GraphCanvas::NodePaletteTreeItem("root", AnimGraphEditorId))
```

Comment:
question: If this is an AZ type then `aznew` is correct but if this does derive from a QWidget or QObject I believe we should just plain `new`

Reply:
Thanks for guidance.

`NodePaletteTreeItem` however doesn't inherit `QObject`. It derives from `GraphCanvas::GraphCanvasTreeItem` which is a plain class:
```cpp
class GraphCanvasTreeItem
{
```